src/main/java/org/mule/module/apikit/validation/attributes/QueryParameterValidator.java

```diff
@@ -121,7 +123,8 @@ private static void validate(String paramKey, Parameter expected, String paramVa
     }
   }
 
-  private static boolean isNullable(Parameter parameter) {
-    return parameter.validate(null);
+  private static boolean isNull(Collection<?> paramValues) {
+    return paramValues == null ||
+        paramValues.size() == 1 && paramValues.stream().allMatch(value -> value == null || "null".equals(value));
```

Comment:
This is my interpretation of the code:
> It `isNull` either if the value for the collection is `null` or if the collection has exactly one value whose value is `null` or "null"

Is this correct? (i'm probably missing some context, sorry for that)
The usage of `.stream()` makes me think that we were expecting more than one value to be tested here, but the restriction `paramValues.size()` explicitly states that we want exactly one parameter to be tested.